dbf2csv/views/views.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
```

Comment:
Transient models / views should go into the `wizards/` directory, as per [OCA guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories).

Reply:
Ok, thanks